### ISSUE TITLE
no longer can assign single letter keyboard shortcuts

### ISSUE THREAD

**Issue by mvolkmann** [Microsoft/vscode#67550]
Issue Type: <b>Bug</b>

Until about a month ago I could use some keyboard shortcuts that were assigned to a single letter. That stopped working. For example:

  {
    "key": "r",
    "command": "renameFile",
    "when": "explorerViewletVisible && filesExplorerFocus && !explorerResourceIsRoot && !inputFocus"
  },

If I change "key" to "ctrl+r" then it works again.
Was there a change to make is so a single letter like "r" cannot be used?

VS Code version: Code - Insiders 1.31.0-insider (2c987b20a4d65ee51c48b9e4eb8f50c926199468, 2019-01-30T06:17:39.118Z)
OS version: Darwin x64 18.0.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (1.22GB free)|
|Process Argv|-psn_0_8820841|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (39)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
bracket-pair-colorizer|Coe|1.0.61
vue-peek|dar|1.0.2
vscode-markdownlint|Dav|0.23.0
vscode-eslint|dba|1.8.0
gitlens|eam|9.4.1
tslint|eg2|1.0.42
vscode-npm-script|eg2|0.3.5
prettier-vscode|esb|1.7.3
flow-for-vscode|flo|1.0.1
auto-close-tag|for|0.5.6
auto-rename-tag|for|0.0.15
lintlens|ghm|2.3.0
fish|gio|0.0.4
vscode-settings-cycler|hoo|1.0.1
vscode-go-to-file|jac|0.0.1
vscode-duplicate|mrm|1.2.1
Go|ms-|0.8.0
vuejs-extension-pack|mub|1.1.3
printcode|nob|3.0.0
vetur|oct|0.14.5
vscode-versionlens|pfl|0.22.0
java|red|0.37.0
vscode-icons|rob|8.0.0
code-spell-checker|str|1.6.10
vscode-status-bar-format-toggle|tom|1.4.0
sort-lines|Tyr|1.7.0
vscodeintellicode|Vis|1.1.2
vscode-java-debug|vsc|0.16.0
vscode-java-dependency|vsc|0.3.0
vscode-java-pack|vsc|0.5.0
vscode-java-test|vsc|0.14.0
vscode-maven|vsc|0.14.1
vim|vsc|1.0.3
quokka-vscode|Wal|1.0.181
vscode-todo-highlight|way|1.0.4
JavaScriptSnippets|xab|1.7.2
ReactSnippets|xab|2.3.0

</details>

**Comment by usernamehw** [comment]
Possible duplicate of #66995

**Comment by alexandrudima** [comment]
This might be caused by the new work in tree filtering.

**Comment by joaomoreno** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/66995

@mvolkmann We're now catching letters in the explorer to make keyboard navigation easier. We'll make it possible for extension to override this behavior. As for your custom keybinding if changing to "ctrl+r" is a possibility I would recommend it. Otherwise, let me know.

**Comment by mvolkmann** [comment]
The reason I want to be able to use “r” to rename files and “d” to delete files (only when the explorer has focus) is that it mirrors how those are done in Vim within netrw. I use the VS Code Vim plugin and these key mappings make it even more like using Vim.

**Comment by joaomoreno** [comment]
Perfect. Today I've submitted https://github.com/VSCodeVim/Vim/pull/3432 which will create a good experience if use the VIM extension: the type filter mode will be triggered only if you press `/`. Just waiting on VIM to merge and release new version and you should be set.
